packages/flutter_tools/lib/src/asset.dart

```diff
@@ -435,6 +483,14 @@ Map<_Asset, List<_Asset>> _parseAssets(
   return result;
 }
 
+_Asset _resolvePackageAsset(
+    String assetBase, String packageName, String asset) {
```

Comment:
Our conventions change over time and we slowly transition the entire codebase over time. Always aim for the most up to date conventions (which ideally are described in the style guide).

You should feel empowered to apply new conventions to old code aggressively. For example I will often pepper trailing commas around the codebase as I'm doing other things.

You should feel empowered to proposed PRs for the Flutter style guide, especially for updating it to match explicit conventions you hear about in reviews, or implicit conventions you observe in the codebase. The style guide isn't complete.